.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit = */migrations/*
+       */fielded_batch_ingest.py
+       */csv_to_pickle.py
+       */dynamic_form.py
+       */batch_valuesets.py
```

Comment:
Gotcha. The only problem with that is that now our coverage reports not correct. We want to omit the migrations because that is code that we did not right, and do not want to test.

I would recommend that we continue to monitor the coverage on those files.